### ISSUE TITLE
Issue TSO Command does not appear to use the TSO profile regionSize parameter

### ISSUE THREAD

**Issue by fswarbrick** [zowe/vscode-extension-for-zowe#1334]
The TSO (IZUFPROC) session created for the "Issue TSO Command" option does not appear to utilize the regionSize parameter of tso-profile.

1. Right click on the appropriate profile under any of the three environments (DATA SETS, USS, JOBS).
2. Select "Issue TSO Command".
3. Enter the TSO command to be executed.
4. After execution, find the generated job in JOBS.
5. View the JES2:JESJCL(3) member
6. Observe that the REGION parameter on the JOB card is set to 4096K, even though the "regionSize" parameter of the default tso-profile is set to another value.

Command should use the appropriate tso-profile regionSize, when defined.

**Desktop (please complete the following information):**

- OS: Windows 10
- Zowe Explorer Version: 1.15.1
- (Optional) Zowe CLI Version: 6.23.0
- (Optional) Are you using Secure Credential Store?  No.

**Additional context**

**Comment by jellypuno** [comment]
@fswarbrick You need to set your mainframe IDs region size to 65536. This is a [z/OSMF requirement](https://docs.zowe.org/v1-1-x/user-guide/systemrequirements-zosmf.html#z-os-requirements)

**Comment by fswarbrick** [comment]
@jellypuno, I don't agree with your interpretation.  That link states that "region size is a minimum of 65536 KB".  Since it is only a minimum, that means a larger value can be specified.  The Zowe CLI "zos-tso" command allows specification of a "--region-size" parameter.  Additionally, one can set up a default tso-profile containing a region size, as well as an account.  The "Issue TSO Command" feature seems to be utilizing the account field, but not the region size field.

**Comment by jellypuno** [comment]
@fswarbrick understood. It seems like you've checked out our code. We do welcome contributions 😊

**Comment by fswarbrick** [comment]
@jellypuno, I have looked at the code, but I have no experience with VSCode extensions, or even JavaScript or TypeScript.  So it would likely take me quite a while to be able to make a good contribution.

Seems like you should be able to get the region size in a manner similar to how the getAccountNumber() method works.  In fact, that probably should be made a more generic method to retrieve both the account number and region size from the TSO profile.

**Comment by jellypuno** [comment]
> I have looked at the code, but I have no experience with VSCode extensions, or even JavaScript or TypeScript. So it would likely take me quite a while to be able to make a good contribution.

The Zowe community is a place of learning. We as a squad are happy to help and guide those who are willing to learn.

As for the prioritization of this issue, this is [our objectives for this PI](https://github.com/zowe/community/blob/master/Project%20Management/PI%20Planning/21PI2%20Planning/Zowe%20Explorer%20Squad%20-%2021PI2%20Objectives%20(May%202021%20to%20July%202021).md). This will be moved to low priority and will be worked on if we have spare time. 

What I could do is add the `help wanted` label so if somebody from the community (including you) would decide to work on this then  they can.

**Comment by fswarbrick** [comment]
So I went ahead and gave this a shot.  I got it to perform the functionality I am looking for.  However some of the unit tests are now failing.  What is my next step?  Do I push my changes, do a pull request, or what?  I am a git novice.
Thanks!

**Comment by fswarbrick** [comment]
@jellypuno Thoughts on my last comment?

**Comment by jellypuno** [comment]
@fswarbrick I am glad. Thank you! Can you open a draft pull request so we can see what the error is and how we could help?

**Comment by jellypuno** [comment]
@fswarbrick apologies if I miss your last comment

**Comment by fswarbrick** [comment]
@jellypuno Draft PR: https://github.com/zowe/vscode-extension-for-zowe/pull/1359
Hopefully I've done things correctly, as this is my first true use of Github.
